src/views/Exchange/Exchange.js

```diff
@@ -477,11 +479,6 @@ export const Exchange = forwardRef((props, ref) => {
 
   const tokens = getTokens(chainId);
 
-  const tokenAddresses = tokens.map((token) => token.address);
```

Comment:
What was this being used for before?

Reply:
It was used for `infoTokens` which I pull in the parent component to share to other components. Thought it would be better than having it created each time. I can restore it to the Exchange component if necessary